docs/DataModels/ToolLayerSchema.md

```diff
@@ -0,0 +1,28 @@
+﻿---
+title: "Tool Layer Schema"
+description: >
+   Extract raw data into a relational schema for each specific tool
+sidebar_position: 2
+---
+
+## Summary
+
+This document describes Apache DevLake's tool layer schema.
+
+Referring to DevLake's [architecture](../Overview/Architecture.md), the Tool layer extracts raw data from JSONs into a relational schema that's easier to consume by analytical tasks. Each DevOps tool would have a schema that's tailored to their data structure, hence the name, the Tool layer.

```

Comment:
grammar: "tailored to their data structure" -> "tailored to _its_ data structure"